includes/admin/class-wc-admin-addons.php

```diff
@@ -32,8 +32,9 @@ public static function get_featured() {
 				$headers['Authorization'] = 'Bearer ' . $auth['access_token'];
 			}
 
+			// Important: WCCOM Extensions API v2.0 is used.
```

Comment:
As mentioned in 11257-gh-Automattic/woocommerce.com#issuecomment-942166271, we want to return the country to the endpoint, like we do on other pages.

```suggestion
			$parameter_string = '';
			$country  = WC()->countries->get_base_country();
			if ( ! empty( $country ) ) {
				$parameter_string = '?' . http_build_query( array( 'country' => $country ) );
			}

			// Important: WCCOM Extensions API v2.0 is used.
```